src/controllers/cities.mjs

```diff
@@ -0,0 +1,88 @@
+import { getFileData, writeFile } from "../utils/fileHelper.mjs";
+import { BASEDIR } from "../../config.mjs";
+import { cityObjValidate } from "../utils/dataValidation.mjs";
+
+const getCities = async () => {
+  return await getFileData(`${BASEDIR}/data/cities.json`);
+};
```

Comment:
In `getAllCities` function you're using this
But in `getCity` you don't

So, lets pick something the same)
I see in another controller that you're getting data directly from fileHelper, so probably I would omit this function (`getCities`) at all